packit_service/worker/build/koji_build.py

```diff
@@ -223,7 +223,7 @@ def run_build(
             task_id = int(task_id_match.group(1))
 
         task_url_match = search(
-            pattern=r"(https://koji\.fedoraproject\.org/koji/taskinfo\?taskID=\d+)",
+            pattern=r"(https://koji\.(stg\.)?fedoraproject\.org/koji/taskinfo\?taskID=\d+)",
```

Comment:
How about using something like below?

```suggestion
            pattern=r"(https://.+/koji/taskinfo\?taskID=\d+)",
```

The above would work with any Koji instance which has a `/koji/taskinfo?taskID=` endpoint.